src/DependencyInjection/JaegerExtension.php

```diff
@@ -18,13 +18,40 @@ public function load(array $configs, ContainerBuilder $container)
             new FileLocator(__DIR__ . '/../Resources')
         );
         $loader->load('services.yml');
-
```

Comment:
please, preserve empty newlines for better readability and visual separation between blocks